### PR TITLE
fix scanner comments

### PR DESCRIPTION
**Pull request by AdrianAndersen**
Fixes the comments from: https://github.com/boklisten/bl-next/pull/345
- refactor(ManualRegistrationModal): loading => waiting
- refactor: unify scanner feedback state
- style: add max width for scanner tutorial
- style: add padding top for tutorial
- style: change close tutorial button color
- fix: close tutorial modal when user clicks outside of it
- style: change close scanner button color
- chore: slightly change info text to cover more cases

### CODE REVIEW

**Comment by AdrianAndersen** [comment]
[Rette opp kommentarer fra merget PR](https://trello.com/c/uAZgwiCA/762-rette-opp-kommentarer-fra-merget-pr)

File: src/components/matches/Scanner/Scanner.tsx [hunk 1]
```diff
@@ -99,10 +104,10 @@ const Scanner = ({ forceUpdate }: { forceUpdate: () => void }) => {
...
       <ScannerTutorial />
       <ScannerFeedback
-        open={feedbackVisible}
-        severity={feedbackSeverity}
-        feedback={feedback}
-        handleClose={() => setFeedbackVisible(false)}
+        open={feedback.visible}
+        severity={feedback.severity}
+        feedback={feedback.text}
+        handleClose={() => setFeedback({ ...feedback, visible: false })}
```
Comment (LarsSelbekk) [inline src/components/matches/Scanner/Scanner.tsx:110]:
```suggestion
        handleClose={() => setFeedback((previous) => ({ ...previous, visible: false }))}
```

Okay I'm sure it has no effect in this instance, but my eyes hurt whenever someone does a state update based on the current state at update-queue time, not at the actual update time 🙃

File: src/components/matches/Scanner/Scanner.tsx [hunk 2]
```diff
@@ -136,7 +141,7 @@ const Scanner = ({ forceUpdate }: { forceUpdate: () => void }) => {
         open={scanModalOpen}
         handleClose={() => {
           setScanModalOpen(false);
-          setFeedbackVisible(false);
+          setFeedback({ ...feedback, visible: false });
```
Comment (LarsSelbekk) [inline src/components/matches/Scanner/Scanner.tsx:144]:
```suggestion
          setFeedback((previous) => ({ ...previous, visible: false }));
```
Ditto

**Comment by LarsSelbekk** [comment]
Still looks pretty bad (actually worse) on small screens (here iPhone 12 Pro)
image

File: src/components/matches/Scanner/ScannerTutorial.tsx [hunk 3]
```diff
@@ -30,6 +34,8 @@ const ScannerTutorial = () => {
             alignItems: "center",
             flexDirection: "column",
             marginTop: "4rem",
+            paddingTop: "1rem",
+            maxWidth: "95%",
```
Comment (LarsSelbekk) [inline src/components/matches/Scanner/ScannerTutorial.tsx:38]: Still looks pretty bad (maybe worse) on e.g. iPhone 12 Pro
image

File: src/components/matches/Scanner/ScannerTutorial.tsx [hunk 4]
```diff
@@ -30,6 +34,8 @@ const ScannerTutorial = () => {
             alignItems: "center",
             flexDirection: "column",
             marginTop: "4rem",
+            paddingTop: "1rem",
```
Comment (LarsSelbekk) [inline src/components/matches/Scanner/ScannerTutorial.tsx:37]: Some marginBottom would also be nice

**Review by LarsSelbekk — CHANGES_REQUESTED** [review]

File: src/components/matches/Scanner/Scanner.tsx (hunk 1 above)
Comment (AdrianAndersen) [reply src/components/matches/Scanner/Scanner.tsx]: Fixed in: https://github.com/boklisten/bl-next/pull/371/commits/f0a8bcf843c36ff791cb14d05f7da05c8bdb8ea1

File: src/components/matches/Scanner/Scanner.tsx (hunk 2 above)
Comment (AdrianAndersen) [reply src/components/matches/Scanner/Scanner.tsx]: https://github.com/boklisten/bl-next/pull/371/commits/f0a8bcf843c36ff791cb14d05f7da05c8bdb8ea1

File: src/components/matches/Scanner/ScannerTutorial.tsx (hunk 3 above)
Comment (AdrianAndersen) [reply src/components/matches/Scanner/ScannerTutorial.tsx]: https://github.com/boklisten/bl-next/pull/371/commits/f0a8bcf843c36ff791cb14d05f7da05c8bdb8ea1

File: src/components/matches/Scanner/ScannerTutorial.tsx (hunk 4 above)
Comment (AdrianAndersen) [reply src/components/matches/Scanner/ScannerTutorial.tsx]: https://github.com/boklisten/bl-next/pull/371/commits/f0a8bcf843c36ff791cb14d05f7da05c8bdb8ea1

**Review by LarsSelbekk — APPROVED** [review]

**Pull request merged**